README.md

```diff
@@ -628,6 +629,14 @@ export const Dialog = styled.dialog`
 `
 ```
 
+### Fonts
+First a few dos and don'ts:  
+- **DON'T** use fonts downloaded from a non-reliable source
```

Comment:
Fonts should *always* be downloaded from the same domain. Loading fonts from a third-party domain will slow down the initial page load because a DNS lookup needs to happen for every domain.

Reply:
What is meant by this: Don't google for Helvetica Neue and download it ;)

Reply:
@ReneDrie that's right but I've seen projects where it would've been perfectly fine to host the fonts on the same domain but a CDN was used anyway.

Loading fonts from an external service will always be slower than loading them from the same domain. Steps 1 and 2 (below) are not necessary when the fonts are loaded from the same domain, but these steps will block the page from rendering when a CDN is used. (Assuming that the font face declarations are in the bundled css file for the site.)

1. Do a DNS lookup
2. Load the css file for the font-faces
3. Load the fonts

I've added an extra line to the list (see next comment).

Reply:
For fonts that are loaded from a different domain, we should use `dns-prefetch` to avoid the DNS lookup issue @leroykorterink is describing.
```html
<link rel="dns-prefetch" href="https://fonts.googleapis.com/" >
```
https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch

Reply:
This is true and there are tons of optimisation steps to do here, but this is completely out of scope for css guidelines.